lib/manager/gitlabci/__fixtures__/gitlab-ci.6.yaml

```
@@ -0,0 +1,11 @@
+image:
+  # comment
+  name: renovate/renovate:19.70.8-slim
+
+services:
+  # comment
+  - name: other/image1:1.0.0
+    alias: imagealias1
+  # another comment
+  - name: other/image2:1.0.0
+    alias: imagealias2
```

```suggestion
  - alias: imagealias2
    name: other/image2:1.0.0
```
This should work too